ui_framework/components/_common_styles.scss

```diff
@@ -15,3 +15,10 @@
 body {
   font-family: $globalFontFamily;
 }
+
+/**
+ * 1. Required for IE11.
+ */
+main {
+  display: block; /* 1 */
+}
```

Comment:
Oh actually, now that this is part of the UI Framework, we need to recompile and commit the CSS. Just run `npm run uiFramework:start` to kick off compilation.